Source/WebCore/rendering/RenderLayerFilters.cpp

```diff
@@ -160,11 +166,14 @@ GraphicsContext* RenderLayerFilters::beginFilterEffect(RenderElement& renderer,
         return nullptr;
 
     auto& filter = *m_filter;
+    auto filterRegion = m_targetBoundingBox;
 
-    // For CSSFilter, filterRegion = targetBoundingBox + filter->outsets()
-    auto filterRegion = targetBoundingBox;
-    if (filter.hasFilterThatMovesPixels())
+    if (is<RenderSVGShape>(renderer))
+        filterRegion = enclosingLayoutRect(m_filter->resolveFilterRegion(renderer, renderer.style().filter(), filterRegion));
```

Comment:
I think that deserves a comment in code, to explain why this overrides the `filterRegion`, instead of using `m_targetBoundingBox`.

Reply:
This code is replaced by calling `RenderSVGShape::currentSVGLayoutRect()` instead.